charts/mirrorbits-parent/values.yaml

```diff
@@ -27,6 +27,44 @@ ingress:
   #     hosts:
   #       - company.org
 
+## Persistent Storage used across components
+storage:
+  ## Manage (and reuse) PV + PVC for all components
+  manage: false
```

Comment:
would `enabled` be better? more consistent with how enabling a feature normally.

Reply:
Fixed by https://github.com/jenkins-infra/helm-charts/pull/774/commits/d38783bdca48a5734850903e119f9e1e6d46d3e8